src/main/resources/init_db.sql

```diff
@@ -0,0 +1,9 @@
+CREATE DATABASE `taxi_db`;
+
+CREATE TABLE `manufacturers` (
```

Comment:
It's a good practice to add here `IF NOT EXISTS` statement.
```suggestion
CREATE TABLE IF NOT EXISTS `manufacturers` (
```